mindsdb/interfaces/datastore/datastore.py

```diff
@@ -100,10 +100,10 @@ def save_datasource(self, name, source_type, source, file_path=None):
             password = self.config['integrations']['default_mariadb']['password']
             host = self.config['integrations']['default_mariadb']['host']
             port = self.config['integrations']['default_mariadb']['port']
-            ds = MariaDS(source, user=user, password=password, host=host, port=port)
+            ds = MariaDS(table='', query=source, user=user, password=password, host=host, port=port)
```

Comment:
@George3d6 please check this. ClickhouseDS above has not 'table' parameter in constructor, but for MariaDS it required. As i assume we should delete 'table' from MariaDB or add it to ClickhouseDS, depending on how GUI will create such datasources.

Reply:
We can leave `table` for MariaDS, I don't think we need to specify it as a parameter, if `query` is specified then `table` is ignored.

Leave it unspecified here please

Reply:
done